docs/tutorials/writing-views.md

```diff
@@ -147,58 +149,70 @@ Here's an example:
 
 ```py
 from zerver.decorator import require_realm_admin
-from zerver.lib.request import has_request_variables, REQ
+from zerver.lib.typed_endpoint import typed_endpoint
 
 @require_realm_admin
-@has_request_variables
-def create_user_backend(request, user_profile, email=REQ(), password=REQ(),
-                        full_name=REQ()):
+@typed_endpoint
+def create_user_backend(
+        request: HttpRequest,
+        user_profile: UserProfile,
+        *,
+        email: str,
+        password: str,
+        full_name: str,
+    ):
     # ... code here
 ```
 
-You will notice the special `REQ()` in the keyword arguments to
-`create_user_backend`. `has_request_variables` parses the declared
-keyword arguments of the decorated function, and for each that has an
-instance of `REQ` as the default value, it extracts the HTTP parameter
-with that name from the request, parses it as JSON, and passes it to
+The `typed_endpoint` decorator parses the declared keyword arguments
```

Comment:
keyword -> keyword-only, and a link to Python's glossary could help: https://docs.python.org/3/glossary.html#term-parameter.